v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java

```diff
@@ -65,11 +67,22 @@
 
         void setInputGcsFilePattern(String inputGcsFilePattern);
 
+        @Description("GCS filepattern for files in bucket to write data to")
+        String getOutputGcsFilePattern();
+
+        void setOutputGcsFilePattern(String outputGcsFilePattern);
+
         @Description("File format of input files. Supported formats: JSON, CSV")
         @Default.Enum("JSON")
         GcsIO.FORMAT getInputGcsFileFormat();
 
-        void setInputGcsFileFormat(GcsIO.FORMAT inputGcsFileFormat);
+        void setInputGcsFileFormat(GcsIO.FORMAT outputGcsFileFormat);
```

Comment:
Seems like `inputGcsFileFormat` shouldn't be replaced with `outputGcsFileFormat`. Or am I missing something?

Reply:
I got too carried away :)